src/lib/expression/placeholder_expression.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "abstract_parameter_expression.hpp"
+
+namespace opossum {
+
+/**
+ * Represents a placeholder (SELECT a + ? ...) in a PreparedPlan. Will replaced by a different expression by
+ * PreparedPlan::instantiate()
+ */
+class PlaceholderExpression : public AbstractParameterExpression {
```

Comment:
As this is an `AbstractParameterExpression`, I'd prefer a name that also ends in `ParameterExpression`